doc-content/enterprise-only/release-notes/rn-731_fixed-issues-ref.adoc

```diff
@@ -0,0 +1,85 @@
+[id='rn-731-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.3.1
+
+{PRODUCT} 7.3.1 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+
+== {CENTRAL}
+
+* The `NoClassDefFoundError` appears in the server log file when a service task is installed in a project [https://issues.jboss.org/browse/RHPAM-1947[RHPAM-1947]]
+* You cannot create a project in an offline environment [https://issues.jboss.org/browse/RHPAM-1282[RHPAM-1282]]
+* {CENTRAL} cannot connect to a co-located controller when deployed at the same time [https://issues.jboss.org/browse/RHPAM-1920[RHPAM-1920]]
+* The package name white list should contain the projects default package [https://issues.jboss.org/browse/RHPAM-1890[RHPAM-1890]]
+* Column sorting is not retained after filters are applied in the Task in box [https://issues.jboss.org/browse/RHPAM-1700[RHPAM-1700]]
+
+== Form modeler
+* When you start a process instance in {CENTRAL} the modal window contains the previously rendered form until the new form renders [https://issues.jboss.org/browse/RHPAM-1951[RHPAM-1951]]
+* The multiple selector throws an error when edited [https://issues.jboss.org/browse/RHPAM-1861[RHPAM-1861]]
+
+== Installer
+* The default placeholder for the controller URL should contain `business-central` when installing {KIE_SERVER} and {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1774[RHPAM-1774]]
```

Comment:
Not sure... shouldn't `business-central` be replaced with a placeholder instead?